packages/storybook/stories/components/text/documentation.mdx

```diff
@@ -75,4 +76,4 @@ The **Text** component is non-interactive and does not receive keyboard focus. I
 
 <Heading label="Accessibility" level={ 2 } />
 
-**Text** content is readable by screen readers.
+This component complies with the <ExternalLink href="https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/Heading_Elements">h1 to h6</ExternalLink>, <ExternalLink href="https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/p">p</ExternalLink>, <ExternalLink href="https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/label">label</ExternalLink>, <ExternalLink href="https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/span">span</ExternalLink>, <ExternalLink href="https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/code">code</ExternalLink>, <ExternalLink href="https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/caption">caption</ExternalLink> HTML elements.
```

Comment:
remove `caption` as it is not actuallay the tag used by the component (it's a `span`)